pkg/skaffold/deploy/helm.go

```diff
@@ -193,8 +193,11 @@ func (h *HelmDeployer) deployRelease(out io.Writer, r v1alpha2.HelmRelease, buil
 		}
 		args = append(args, "-f", constants.HelmOverridesFilename)
 	}
-	if r.ValuesFilePath != "" {
-		args = append(args, "-f", r.ValuesFilePath)
+
+	if len(r.ValuesFilePath) != 0 {
```

Comment:
The `if` can be skipped safely here since the loop will do nothing if there's no value